src/vcf.rs

```diff
@@ -203,7 +149,7 @@ impl VCFFile {
                 }
             }
             // Add minor calls if the filter is passed or ignored, or specifcally just the MIN_FRS has failed
-            if ignore_filter || passed || filters.contains(&"MIN_FRS".to_string()) {
+            if ignore_filter || passed || record.filter.contains(&"MIN_FRS".to_string()) {
```

Comment:
Based on the comment string the last check should be something like `record.filter == vec![&"MIN_FRS".to_string()]`

Reply:
Ah good catch. I think it actually needs to match the filters in the allowed filters list above